### DISCUSSION TITLE
Is it possible to add link explore that will have links to all pages?

### DISCUSSION THREAD

**Discussion by jcubic** [EddieHubCommunity/LinkFree#1351] [Q&A] [answered]
It would be nice to make actual use of this project for linking to have a page like "explore" that will show all profiles (possible with paginations) so they actually can be indexed by google. Right now I only see search. You can do a blank search but this is not the same as having link to profiles.

PS: [Your project is not indexed by Google](https://www.google.com/search?hl=pl&q=site%3Ainkfree.eddiehub.io).

**Comment by NS007-dev** [comment, marked as answer]
@jcubic I think it is indexed by google because you have made a typo in the search:
![image](https://user-images.githubusercontent.com/70807500/171485978-94898d64-f3c5-4f8c-9359-a22fe0e58a62.png)

you put `site:inkfree.eddiehub.io`
but it should be `site:linkfree.eddiehub.io`.

Thank you.
